### PR TITLE
Pr/backlog cleanup

### PR DESCRIPTION
**Pull request by jordonezlucena**

### CODE REVIEW

**Comment by jordonezlucena** [comment]
[#104](https://github.com/camaraproject/Governance/pull/104)
#332

**Comment by jordonezlucena** [comment]
Both PRs have been merged, so this issue is eligible to be closed @hdamker

**Comment by hdamker** [comment]
@jordonezlucena 

> Both PRs have been merged, so this issue is eligible to be closed @hdamker

This here is a PR which need to be reviewed and merged, not an issue which can be closed. 

https://github.com/camaraproject/Governance/pull/104 and https://github.com/camaraproject/WorkingGroups/pull/332 were related to the movement of the API Onboarding document, not to the cleanup of the table

**Comment by hdamker** [comment]
@jordonezlucena Thanks a lot for the work to clean-up the table, I know how tedious it is to edit this table in markdown (I only tried to do three lines of it).

That's also the reason that I'm very sorry, but I'm not able to review it, as the table is completely replaced and therefore github is not showing the changes anymore. And "View file" shows it in a way which can't be reviewed either (scrolling in two directions necessary, always only showing a small part of it). I Agree that his is a general problem, but it seems that it won't work in this way.

On point I have seen: it seems that the PR contains changes which are going beyond a clean-up, e.g. the change of "number verify" to "number_verification" or the addition of the new proposals (at least one pointing to a closed PR) - such changes I would recommend to do seperately for transparency reasons.

**Comment by jordonezlucena** [comment]
> @jordonezlucena Thanks a lot for the work to clean-up the table, I know how tedious it is to edit this table in markdown (I only tried to do three lines of it).

> That's also the reason that I'm very sorry, but I'm not able to review it, as the table is completely replaced and therefore github is not showing the changes anymore. And "View file" shows it in a way which can't be reviewed either (scrolling in two directions necessary, always only showing a small part of it). I Agree that his is a general problem, but it seems that it won't work in this way.

Yes, this PR only plans to cover the first activity among the ones listed in [#100](https://github.com/camaraproject/Governance/issues/100). We then need to decide which information will be migrated to Wiki (and being evolved therein) and which content will be kept in the API backlog.md file. We can also decide whether API backlog.md file should be kept as .md file or if it makes sense to migrate it to an HTML file.

> On point I have seen: it seems that the PR contains changes which are going beyond a clean-up, e.g. the change of "number verify" to "number_verification" or the addition of the new proposals (at least one pointing to a closed PR) - such changes I would recommend to do separately for transparency reasons.

As to the number verification, I've taken as a reference the [latest AHC report](https://github.com/camaraproject/Governance/blob/main/documentation/MeetingMinutes/AHC/CAMARA%20Project%20-%20AHC%20-%202023-10-12.pdf), which states the Sub-Project name and API name are 'NumberVerification' and 'number_verification.yaml'. That's something pointed out in [#100](https://github.com/camaraproject/Governance/issues/100). 
As to the addition of new proposals, I can keep doing in the main file (no further commitment to this PR), though there may be conflicts when merging. With regards to the closed PR, it's something I may have forgotten to update it. Could you please  drop me a comment on the specific row where you have identified the mistake, so I can fix it accordingly?

File: APIBacklog/documentation/APIBacklog.md [hunk 1]
```diff
@@ -3,51 +3,61 @@
...
+ |  | | | | |  | | |  | v0.8.1 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.1/code/API_definitions) | | | DTAG | DTAG - Germany |
+|  | | |   |  | | |  | v0.8.0 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.0/code/API_definitions) | | [PI_1 (DT)](https://github.com/camaraproject/QualityOnDemand_PI1)<BR>[PI_3 (SFN)](https://github.com/camaraproject/QualityOnDemand_PI3)| DTAG | DTAG - Germany |<!---Simple Edge Discovery --->
+| Simple Edge Discovery | Vodafone | N/A | N/A  | 5GFF, Vodafone | Approved (N/A)  | [EdgeCloud/Discovery](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Discovery) | simple-edge-discovery | [Link](https://github.com/camaraproject/EdgeCloud/tree/main/documentation) | v0.9.1 | [YAML](https://github.com/camaraproject/EdgeCloud/blob/main/code/API_definitions/Discovery/simple_edge_discovery.yaml) | N/A | N/A | N/A| N/A|<!---MEC Management --->
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:18]: The link is pointing to main, not to a release branch or tagged commit. 
There is a tagged version v0.8.1 https://github.com/camaraproject/EdgeCloud/blob/v0.8.1/code/API_definitions/simple_edge_discovery.yaml

File: APIBacklog/documentation/APIBacklog.md [hunk 2]
```diff
@@ -3,51 +3,61 @@
...
+| MEC Management | Telefonica | N/A | N/A  | GSMA-OGW, Telefonica | Approved (N/A) |[EdgeCloud/Discovery](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Discovery)  | mec-exposure-and-experience-management | [Link](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Discovery) | v0.8.1 | [YAML](https://github.com/camaraproject/EdgeCloud/blob/main/code/API_definitions/Discovery/MEC%20exposure%20and%20experience%20management.yaml) | N/A | N/A | N/A| N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Device Location-->%%%%%%%%%%%%%%%%%%%%%
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:19]: Same as above. There is a tagged version v0.8.1 https://github.com/camaraproject/EdgeCloud/blob/v0.8.1/code/API_definitions/MEC%20exposure%20and%20experience%20management.yaml

File: APIBacklog/documentation/APIBacklog.md [hunk 3]
```diff
@@ -3,51 +3,61 @@
...
+| Device Location | DTAG | 2022/01/11 | N/A | DTAG, Orange, Telefonica, Vodafone | Approved (2022/04/17) | [DeviceLocation](https://github.com/camaraproject/DeviceLocation) | location-verification | [Link](https://github.com/camaraproject/DeviceLocation/blob/main/documentation) | v0.2.0-wip | [YAML](https://github.com/camaraproject/DeviceLocation/blob/main/code/API_definitions/location-verification.yaml) | N/A | N/A | N/A|
+| |  |  |  |  |   | |  |  | v0.1.0 | [YAML](https://github.com/camaraproject/DeviceLocation/blob/release-v0.1.0/code/API_definitions/location.yaml) | N/A | N/A | N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Device Status-->%%%%%%%%%%%%%%%%%%%%%
+| Device Status | DTAG | 2022/01/11 | N/A | DTAG, Orange, Telefonica, Vodafone | Approved (2022/04/17) |  [DeviceStatus](https://github.com/camaraproject/DeviceStatus) | device-status | [Link](https://github.com/camaraproject/DeviceStatus/tree/main/documentation) | v0.5.0-wip | [YAML](https://github.com/camaraproject/DeviceStatus/blob/main/code/API_definitions/device-status.yaml) |  N/A |  N/A |  N/A|
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:22]: Same as above, this is not a released version yet.

File: APIBacklog/documentation/APIBacklog.md [hunk 4]
```diff
@@ -3,51 +3,61 @@
...
+
+
+
+## API backlog
+
+ This is a live doc that captures the status of all the APIs which have been formally registered in CAMARA project. 
+
+ | **API [family] name**  |  **API [family] owner** |  **API [family] proposal registration date**  | **Link to the filled-in application template**   |  **Supporters in API Backlog Working Group**  |  **TSC decision (date)** |  **Sub Project**  |  **API name**  |  **API documentation** |  **API version**  | **API definition**  |  **Implementation**  |  **API tested at** |  **API deployed at company and country**  |  
+ | --- |  --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |---| --- |
+ | Quality on Demand | DTAG | N/A | N/A | DTAG, Orange, Telefonica, Vodafone|  Approved (N/A) | [QualityOnDemand](https://github.com/camaraproject/QualityOnDemand) | qod | [Link](https://github.com/camaraproject/QualityOnDemand/tree/main/documentation/API_documentation) | v0.9.0 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.9.0/code/API_definitions) | [PI_2 (Orange)](https://github.com/camaraproject/QualityOnDemand_PI2) | DTAG |  N/A |
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:15]: The documentation of the API is inline within the YAML, not within the linked directory. And: the documentation links should also point to the release branch to show the documentation which belongs to the released version.

```suggestion
 | Quality on Demand | DTAG | N/A | N/A | DTAG, Orange, Telefonica, Vodafone|  Approved (N/A) | [QualityOnDemand](https://github.com/camaraproject/QualityOnDemand) | qod | [Inline](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/release-0.9.0/code/API_definitions/qod-api.yaml) | v0.9.0 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.9.0/code/API_definitions) | [PI_2 (Orange)](https://github.com/camaraproject/QualityOnDemand_PI2) | DTAG |  N/A |
```

File: APIBacklog/documentation/APIBacklog.md [hunk 5]
```diff
@@ -3,51 +3,61 @@
...
+ |  | | | | |  | | |  | v0.8.1 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.1/code/API_definitions) | | | DTAG | DTAG - Germany |
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:16]: Suggest to add the documentation which belongs to the release:

```suggestion
 |  | | | | |  | | | [Link](https://github.com/camaraproject/QualityOnDemand/blob/release-0.8.1/documentation/API_documentation/) | v0.8.1 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.1/code/API_definitions) | | | DTAG | DTAG - Germany |
```

File: APIBacklog/documentation/APIBacklog.md [hunk 6]
```diff
@@ -3,51 +3,61 @@
...
+| Device Status | DTAG | 2022/01/11 | N/A | DTAG, Orange, Telefonica, Vodafone | Approved (2022/04/17) |  [DeviceStatus](https://github.com/camaraproject/DeviceStatus) | device-status | [Link](https://github.com/camaraproject/DeviceStatus/tree/main/documentation) | v0.5.0-wip | [YAML](https://github.com/camaraproject/DeviceStatus/blob/main/code/API_definitions/device-status.yaml) |  N/A |  N/A |  N/A|
+| |  |  |  |  |  |   |  |  | v0.4.1 |  N/A | N/A | N/A | N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Home Devices Quality on Demand --->
+| Home Devices Quality on Demand | Telefonica | 2022/09/14 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_HomeDevicesQoD_Telefonica.md) | DTAG, KDDI, Telefonica |  Approved (2022/10/13) | [HomeDevicesQoD](https://github.com/camaraproject/HomeDevicesQoD) | home_devices_qod | [Link](https://github.com/camaraproject/HomeDevicesQoD/blob/main/documentation/API_documentation) | v0.2.0 | [YAML](https://github.com/camaraproject/HomeDevicesQoD/blob/main/code/API_definitions/home_devices_qod.yaml) |  N/A |  N/A |  N/A|
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:24]: Links should point to the release branch of v0.2.0, not to main. The API has inline documentation, link from the release notes: https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/HomeDevicesQoD/release-0.2.0/code/API_definitions/home_devices_qod.yaml

File: APIBacklog/documentation/APIBacklog.md [hunk 7]
```diff
@@ -3,51 +3,61 @@
...
+| | |  | | | | | | | v0.1.0 | [YAML](https://github.com/camaraproject/HomeDevicesQoD/blob/release-0.1.0/code/API_definitions/home_devices_qod.yaml) |  N/A |  N/A |  N/A|<!---SIMSwap-->%%%%%%%%%%%%%%%%%%%%%
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:25]: Same as above, release v0.1.0 has a related documentation link: https://github.com/camaraproject/HomeDevicesQoD/tree/release-0.1.0/documentation/API_documentation

File: APIBacklog/documentation/APIBacklog.md [hunk 8]
```diff
@@ -3,51 +3,61 @@
...
+| SIM Swap | DTAG | 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_Sim%20Swap_DeustcheTelekom.md) | Airtel, DTAG, GSMA-IDG, KDDI, Teelfonica | Approved (2022/10/13) | [SIMSwap](https://github.com/camaraproject/SimSwap) | sim_swap | [Link](https://github.com/camaraproject/SimSwap/tree/main/documentation) | v0.4.1 | [YAML](https://github.com/camaraproject/SimSwap/blob/main/code/API_definitions/sim_swap.yaml) | N/A | N/A | N/A|N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Number Verification-->%%%%%%%%%%%%%%%%%%%%%
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:26]: There is no released or tagged version ... the link into main points in the meantime to 0.5.0-wip

File: APIBacklog/documentation/APIBacklog.md [hunk 9]
```diff
@@ -3,51 +3,61 @@
...
+| Number Verification | DTAG| 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_%20NumberVerification_DeustcheTelekom.md) | Airtel, DTAG, Orange, Telefonica | Approved (2022/10/13) | [NumberVerification](https://github.com/camaraproject/NumberVerification/tree/main) | number_verification | [Link](https://github.com/camaraproject/NumberVerification) | v0.4.0-wip | [YAML](https://github.com/camaraproject/NumberVerification/blob/main/code/API_definitions/number_verification.yaml) | N/A | N/A | N/A|N/A|N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---OTPValidation-->%%%%%%%%%%%%%%%%%%%%%
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:27]: Same as above, this is not a released version yet.

File: APIBacklog/documentation/APIBacklog.md [hunk 10]
```diff
@@ -3,51 +3,61 @@
...
+| OTP Validation | DTAG| 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_NumberVerificationMS2FA_DeustcheTelekom.md) | Airtel, DTAG, GSMA-IDG, KDDI, Telefonica  | Approved (2022/10/13) | [OTPValidationAPI](https://github.com/camaraproject/OTPValidationAPI) | one-time-password-sms | [Link](https://github.com/camaraproject/OTPvalidationAPI/blob/main/documentation/API_documentation) | v0.5.0 | [YAML](https://github.com/camaraproject/OTPvalidationAPI/blob/main/code/API_definitions/one-time-password-sms.yaml) | N/A | N/A | N/A|N/A|%%%%%%%%%%%%%%%%<!---Carrier Billing CheckOut-->%%%%%%%%%%%%%%%%%%%%%
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:28]: Links are pointing to main, not the release branch (also wrong in the sub project README, I opened an issue there).
YAML has inline documentation.

File: APIBacklog/documentation/APIBacklog.md [hunk 11]
```diff
@@ -3,51 +3,61 @@
...
+| Carrier Billing Checkout | Telefonica | 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_CarrierBillingCheckOut_Telefonica.md)| Airtel, GSMA-OGW, KDDI, Orange, Telefonica | Approved (2022/10/13) | [CarrierBillingCheckOut](https://github.com/camaraproject/CarrierBillingCheckOut)  | Carrier Billing Check Out| [Link](https://github.com/camaraproject/CarrierBillingCheckOut/blob/main/documentation/API_documentation) | v0.2.0 | [YAML](https://github.com/camaraproject/CarrierBillingCheckOut/blob/main/code/API_definitions/carrier_billing.yaml) | N/A| N/A | N/A|<!---Traffic Influence --->
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:29]: There is a release tag v0.5.0? ... but in main the carrier_billing.yaml says v0.2.0-wip?
We need to clarify this.

File: APIBacklog/documentation/APIBacklog.md [hunk 12]
```diff
@@ -3,51 +3,61 @@
...
+| Traffic Influence | Telecom Italia | 2022/11/02 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_Traffic%20Influence_TIM.md) | GSMA-OPAG, Telecom Italia | Approved (2022/11/10) | [EdgeCloud/TrafficInfluence](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Traffic%20Influence) | traffic-influence | [Link](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Traffic%20Influence) | v0.8.1| [YAML](https://github.com/camaraproject/EdgeCloud/blob/main/code/API_definitions/Traffic%20Influence/Traffic_Influence.yaml) | N/A | N/A | N/A | %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Blockchain Public Address-->%%%%%%%%%%%%%%%%%%%%%
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:30]: Links should point to the release tag

File: APIBacklog/documentation/APIBacklog.md [hunk 13]
```diff
@@ -3,51 +3,61 @@
...
+| Blockchain Public Address | Telefonica | 2023/01/19| [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_BlockchainPublicAddress_Telefonica.md)| Orange, STC, Telefonica| Approved (2023/06/15)  | [BlockchainPublicAddress](https://github.com/camaraproject/BlockchainPublicAddress)  | Blockchain Public Address | [Link](https://github.com/camaraproject/BlockchainPublicAddress/tree/main/documentation/API_documentation) | v0.1.0 | [YAML](https://github.com/camaraproject/BlockchainPublicAddress/blob/main/code/API_definitions/blockchain_public_address.yaml) | N/A| N/A | N/A |%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Identity and Consent Management-->%%%%%%%%%%%%%%%%%%%%%
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:31]: v0.1.0 is only "baseline contribution" and main branch is not any more the tagged v0.1.0

File: APIBacklog/documentation/APIBacklog.md [hunk 14]
```diff
@@ -3,51 +3,61 @@
...
+| Device Identifier | Vodafone | 2023/05/12 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_DeviceIdentifier_Vodafone.md) | DTAG, KDDI, Telefonica, Vodafone | Approved (2022/10/13) | [DeviceIdentifier](https://github.com/camaraproject/DeviceIdentifier) | Device Identifier | [Link](https://github.com/camaraproject/DeviceIdentifier/tree/main/documentation) | v0.0.3 | [YAML](https://github.com/camaraproject/DeviceIdentifier/blob/main/code/API_definitions/CAMARA%20Device%20Identifier%20API.yaml) | N/A | N/A | N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Click-to-Dial-->%%%%%%%%%%%%%%%%%%%%%
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:33]: There is no release yet. Main shows currently v0.1.0 but I wouldn't list that.

File: APIBacklog/documentation/APIBacklog.md [hunk 15]
```diff
@@ -3,51 +3,61 @@
...
+| Network Insights | Verizon | 2023/08/31| [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_NetworkInsights_Verizon.md) | 5GFF, GSMA-OGW, Verizon | Approved (2023/10/19) | N/A | N/A | N/A | N/A | N/A | N/A | N/A| N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Network Slicing-->%%%%%%%%%%%%%%%%%%%%%
+| Network Slicing | China Unicom | 2023/09/26| [Link](https://github.com/camaraproject/WorkingGroups/pull/316) |China Unicom | Under evaluation (YYYY/MM/DD)| N/A | N/A | N/A | N/A | N/A | N/A | N/A|N/A |%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---KYC Age verification-->%%%%%%%%%%%%%%%%%%%%%
```
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:42]: https://github.com/camaraproject/WorkingGroups/pull/316 is closed.
Replacement seems to be https://github.com/camaraproject/WorkingGroups/pull/333

File: APIBacklog/documentation/APIBacklog.md (hunk 11 above)
Comment (hdamker) [inline APIBacklog/documentation/APIBacklog.md:29]: API name seems to be in the meantime "Carrier Billing"?

**Review by hdamker — COMMENTED** [review]
"Display the rich diff" allowed to see the changes ... see my suggestions for changes (especially the need to link the releases, not main in several cases).

**Comment by jordonezlucena** [comment]
PR replaced by #360.

**Pull request closed without merging**